dev/com.ibm.ws.transport.http2_fat/test-applications/H2FATDriver.war/src/http2/test/driver/war/servlets/H2FATDriverServlet.java

```diff
@@ -1926,10 +1926,10 @@ public void testInvalidHeaderBlock(HttpServletRequest request,
         Http2Client h2Client = getDefaultH2Client(request, response, blockUntilConnectionIsDone);
 
         if (USING_NETTY) {
-//            FrameRstStream rstFrame = new FrameRstStream(3, PROTOCOL_ERROR, false);
-//            h2Client.addExpectedFrame(rstFrame);
-            FrameGoAway errorFrame = new FrameGoAway(0, "Protocol error: Padding should not be larger than payload length.".getBytes(), PROTOCOL_ERROR, 2147483647, false);
-            h2Client.addExpectedFrame(errorFrame);
+            FrameRstStream rstFrame = new FrameRstStream(3, PROTOCOL_ERROR, false);
+            h2Client.addExpectedFrame(rstFrame);
```

Comment:
Create a Netty issue to discuss why this would be a protocol error and reset instead of a connection error